### PR TITLE
[TGL] Initial platform package baseline for TGL platform support

### PR DESCRIPTION
**Pull request by jinjhuli**
- Build command
  python BuildLoader.py build tgl

- Stitch command
  python Platform/TigerlakeBoardPkg/Script/StitchLoader.py
  -i <EXISTING_IFWI_IMAGE>
  -o <SBL_IFWI_IMAGE>
  -s Outputs/tgl/SlimBootloader.bin

Signed-off-by: jinjhuli <jin.jhu.lim@intel.com>

### CODE REVIEW

File: Platform/TigerlakeBoardPkg/AcpiTables/AcpiTablePlatform.h [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*++
+
+Copyright (c)  1999  - 2019, Intel Corporation. All rights reserved
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent are licensed and made available under
+
```
Comment (gdong1) [inline Platform/TigerlakeBoardPkg/AcpiTables/AcpiTablePlatform.h:6]: Please also check other files. It looks "are licensed and made available under" should be removed to make the line looks as below: 
SPDX-License-Identifier: BSD-2-Clause-Patent

File: Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  Include ("Cpu.asl")
+  Include ("PciDrc.asl")
+  Include ("Video.asl")
+  Include ("PlatformGnvs.asl")
+  Include ("FwuWmi.asl")
+  Include ("Gpe.asl")
+
+  Include ("PchRpPxsxWrapper.asl")
+  Include ("WifiDynamicSar.asl")
+  Include ("Pep.asl")
```
Comment (gdong1) [inline Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl:184]: I remember you cleaned up some asl code, please make sure you uploaded the cleaned up one.

File: Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+  Include ("PinDriverLib.asl")
+  Include ("Pld.asl")
+
+  If(LEqual(CVFS,1)) {
+    Include("Cvf.asl")
+  }
+
+  //If(LEqual(VMDE,1)) {
+   // Include("Vmd.asl")
+  //}
```
[the remark this answers is not in the document]
Comment (jinjhuli) [reply Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl:206]: Done

File: Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+  Include ("MipiCamSensors.asl")
+  Include ("SerialIoDevices.asl")
+  Include ("SerialIoTimingParametersEmbedded.asl")
+
+  Include ("HdaSlaveDevices.asl")
+  Include ("HdaDspModules.asl")
+  Include ("HidPlatformEventDev.asl")
+//#if FixedPcdGetBool(PcdEcEnable) == 1
+  Include ("HIDWakeDSM.asl")
+//#endif
```
[the remark this answers is not in the document]
Comment (jinjhuli) [reply Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl:196]: Done

File: Platform/TigerlakeBoardPkg/AcpiTables/AcpiTablePlatform.h (hunk 1 above)
Comment (jinjhuli) [reply Platform/TigerlakeBoardPkg/AcpiTables/AcpiTablePlatform.h:6]: Done

File: Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl (hunk 2 above)
Comment (jinjhuli) [reply Platform/TigerlakeBoardPkg/AcpiTables/Dsdt/Dsdt.asl:184]: Done

**Review by gdong1 — APPROVED** [review]

**Pull request merged**